plugins/openwrt/snmp__memory_openwrt

```diff
@@ -1,76 +1,81 @@
-#!/usr/bin/perl -w
-#
-# Copyright (C) 2008 J.M.Roth
-#
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; version 2 dated June,
-# 1991.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
-#
-#
-# $Log$
-#
-# Inspired by snmp__processes
-#
-#%# family=snmpauto
-#%# capabilities=snmpconf
+#!/usr/local/bin/perl -w
```

Comment:
I am not aware of the specific situation of perl in openwrt, but I guess, it is installed in `/usr/bin/perl`?
Or is there a good reason to change this path?